howtos/diversification.rst

```diff
@@ -0,0 +1,37 @@
+Diversification of Search Results
+=================================
+
+.. contents:: Table of contents
+
+Introduction
+------------
+
+Xapian allows for diversification of documents which are stored in the form of an MSet.
+This feature is a well-known technique in information retrieval used to increase
+user satisfaction, especially for ambiguous queries.
+
+Xapian currently has an implementation of an *implicit* method (using documents as features,
+as opposed to using query based features such as query logs) adapted from the C :sup:`2` - GLS method mentioned in Scalable and Efficient Web Search Results Diversification, Naini et al. 2016. This saves the cost of not having to provide external features such as query
+logs, while still achieving the desired diversification effect, which according to
```

Comment:
I'd recommend breaking the sentence before "which", and starting the next one "According to the paper, the effect obtained using this implicit method is reasonable enough…", or something similar. Once sentences go above a couple of clauses, they become a lot harder to figure out. The important information is in the start of this sentence, so breaking out the experimental evidence that we've chosen a good approach into its own sentence should help people read and understand.

Reply:
I've pushed a commit to address this.